modules/wallet/money.go

```diff
@@ -13,17 +13,25 @@ type sortedOutputs struct {
 	outputs []types.SiacoinOutput
 }
 
+// managedDustThreshold returns the quantity below which a Currency is considered to be Dust.
+func (w *Wallet) managedDustThreshold() types.Currency {
+	minFee, _ := w.tpool.FeeEstimation()
+	return minFee.Mul64(3)
```

Comment:
Why does this need a special helper function? Should we follow the pattern discussed in the (defragThreshold PR)[https://github.com/NebulousLabs/Sia/pull/2242#discussion-diff-133795634R49] and inline this?

Reply:
Ups, I did not see that. I still think the helper is useful as it is called 4 times in 3 different files and `dustThreshold` is seen as an entity as issues #2291 & #2292 point. But you decide.

Reply:
I think in this case it probably makes sense to have a separate helper, since the `dustThreshold` value is useful in multiple places throughout the wallet.

Reply:
Unlike the fee calculations, which will always be using the `FeeEstimation` from the tpool, the dust threshold doesn't need to be inherently 3x the minFee. That's what it is in Bitcoin, but you could make a case for 1.5x as well, and potentially you could even have a more elaborate situation that takes into account historic fee lows, using some other method than FeeEstimation.

So the TL;DR is that I'm not really confident that this is the optimal way to do the dust threshold,  but I am confident that doing it this way is better than the way we were previously doing it.

Reply:
For example, if we make the dust threshold more of a long-term value (2x the smallest value seen in the past 12,000 blocks), then we could have the defrag function refuse to run if the `FeeEstimation` is not close enough to the cost of performing the defrag on that particular output. That makes the defragger smarter, but also requires altering this function in a way that is relevant to everyone else using it.